index.php

```diff
@@ -2,12 +2,12 @@
 /**
  * Plugin Name:     The Events Calendar Extension: Relabeler
  * Description:     Adds option to WP Admin > Events > Display for altering labels. For example, you can change the word "Events" to a different word such as "Gigs".
- * Version:         1.0.1
+ * Version:         1.0.2
  * Extension Class: Tribe__Extension__Relabeler
  * Author:          Modern Tribe, Inc.
  * Author URI:      http://m.tri.be/1971
- * License:         GPLv2 or later
- * License URI:     https://www.gnu.org/licenses/gpl-2.0.html
+ * License:         GPLv3 or later
+ * License URI:     ttps://www.gnu.org/licenses/gpl-3.0.html
  */
```

Comment:
And generate a `.pot` in the `languages` folder

```suggestion
 * Text Domain:       tribe-ext-relabeler
 */
```